package/Codegen/TSTypes.cs

```diff
@@ -18,8 +18,8 @@ namespace Codegen
     /// Class to produce the template output
     /// </summary>
     
-    #line 1 "F:\react-native-xaml\package\Codegen\TSTypes.tt"
-    [global::System.CodeDom.Compiler.GeneratedCodeAttribute("Microsoft.VisualStudio.TextTemplating", "16.0.0.0")]
+    #line 1 "C:\Users\asklar\source\repos\react-native-xaml\package\Codegen\TSTypes.tt"
```

Comment:
Do you want your name coded in here?

Reply:
yeah this is autogenerated code, the `#line` directive is just there for help debugging